src/BoundingBox.cpp

```diff
@@ -47,7 +47,9 @@ glm::vec3 BoundingBox::calcReflNormal(BoundingBox& other) {
    }
    if((this->min_.y - epsilon <= other.max_.y && this->min_.y + epsilon >= other.max_.y) ||
            (this->max_.y - epsilon <= other.min_.y && this->max_.y + epsilon >= other.min_.y)){
-       normal = glm::vec3(0.0, 1.0, 0.0);
+
+       // TODO(rgarmsen2295): Hack, should be y as normal, figure out why collisions are matching x instead of y only
+       normal = glm::vec3(1.0, 0.0, 0.0);

```

Comment:
Could the hedges be rotated (or lack rotation)? Do we know the initial orientation of `cube.obj` it's possible it is rotated 90 degrees along the z-axis or something? I dunno if this would actually effect anything if we are computing the `y` value regardless or object orientation. I didn't really play around with the bounding box stuff.